### PR TITLE
Use src and dist folders

### PR DESCRIPTION
**Pull request by thetayloredman**
**Does this Pull Request fix an issue? If so, please mention the issue.**
N/A

**Describe your changes.**
**MAJOR CHANGE!** All code is now in src/ and outputs to dist/! ***This PR might have trouble merging with upstream!***

**Describe alternatives you've considered**
None

**Additional context**
None

### CODE REVIEW

**Comment by thetayloredman** [comment]
Problem: TS doesn't like module-aliases like this.

File: src/log.ts [hunk 1]
```diff
@@ -39,28 +39,28 @@ try {
...
-    warnStr = fs.createWriteStream(`./logs/${logInitTime}/warn.log`);
+    warnStr = fs.createWriteStream(`../logs/${logInitTime}/warn.log`);
 } catch (e) {
     console.error(e);
     process.exit(1);
 }
 let errStr: fs.WriteStream | null = null;
 try {
-    errStr = fs.createWriteStream(`./logs/${logInitTime}/err.log`);
+    errStr = fs.createWriteStream(`../logs/${logInitTime}/err.log`);
```
Comment (thetayloredman) [inline src/log.ts:63]: Should we do logs in the root or maybe some sort of /debug/ directory?

**Review by DeveloperDrago — APPROVED** [review]

**Review by DeveloperDrago — APPROVED** [review]
Looks good from the looks of it

File: src/log.ts (hunk 1 above)
Comment (thetayloredman) [reply src/log.ts:63]: This actually sounds right...

**Pull request merged**